src/tests/receiving/assertCreationOfReceivingBin.test.ts

```diff
@@ -0,0 +1,135 @@
+import { ShipmentType } from '@/constants/ShipmentType';
+import { expect, test } from '@/fixtures/fixtures';
+import CreateLocationPage from '@/pages/location/createLocation/CreateLocationPage';
+import LocationListPage from '@/pages/location/LocationListPage';
+import { StockMovementResponse } from '@/types';
+
+test.describe('Assert creation of receiving bin', () => {
+  let STOCK_MOVEMENT: StockMovementResponse;
+
+  test.beforeEach(
+    async ({
+      supplierLocationService,
+      stockMovementService,
+      mainProductService,
+      otherProductService,
+    }) => {
+      const supplierLocation = await supplierLocationService.getLocation();
+      const PRODUCT_ONE = await mainProductService.getProduct();
+      const PRODUCT_TWO = await otherProductService.getProduct();
+
+      STOCK_MOVEMENT = await stockMovementService.createInbound({
+        originId: supplierLocation.id,
+      });
+
+      await stockMovementService.addItemsToInboundStockMovement(
+        STOCK_MOVEMENT.id,
+        [
+          {
+            productId: PRODUCT_ONE.id,
+            quantity: 100,
+          },
+          { productId: PRODUCT_TWO.id, quantity: 10 },
+        ]
+      );
+
+      await stockMovementService.sendInboundStockMovement(STOCK_MOVEMENT.id, {
+        shipmentType: ShipmentType.AIR,
+      });
+    }
+  );
+
+  test.afterEach(async ({ stockMovementShowPage, stockMovementService }) => {
+    await stockMovementShowPage.goToPage(STOCK_MOVEMENT.id);
+    await stockMovementShowPage.rollbackButton.click();
+    await stockMovementService.deleteStockMovement(STOCK_MOVEMENT.id);
+  });
+
+  test('Assert receiving bin is not created when shipment is shipped', async ({
+    stockMovementShowPage,
+    receivingPage,
+    page,
+    locationListPage,
+    mainLocationService,
+    createLocationPage,
+    browser,
+  }) => {
+    await test.step('Go to stock movement show page and assert shipped status', async () => {
+      await stockMovementShowPage.goToPage(STOCK_MOVEMENT.id);
+      await stockMovementShowPage.isLoaded();
+      await expect(stockMovementShowPage.statusTag).toHaveText('Shipped');
+    });
+
+    await test.step('Go to Bin location tab of edit location page', async () => {
+      const mainLocation = await mainLocationService.getLocation();
+      await page.goto('./location/list');
+      await locationListPage.searchByLocationNameField.fill(mainLocation.name);
+      await locationListPage.findButton.click();
+      await expect(
+        locationListPage.getLocationEditButton(mainLocation.name)
+      ).toBeVisible();
+      await locationListPage.getLocationEditButton(mainLocation.name).click();
+      await createLocationPage.binLocationTab.click();
+      await createLocationPage.binLocationTabSection.isLoaded();
+    });
+
+    await test.step('Assert Bin location is not created yet', async () => {
+      const receivingBin = 'R-' + STOCK_MOVEMENT.identifier;
```

Comment:
I think it would be worth to move `R-` to a global const or even an environment variable, because this is configurable, and once changed on the OB system config, you'd need to update it everywhere.

Reply:
@kkrawczyk123 I think you can add it to the `.env` file, and load it in the `AppConfig` or `LocationConfig`, and there you can load it from the env with a fallback to `R-`